tests/test_getting_started.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -eo pipefail
+
+# Step 1: Python environment
+# Check Python version is or later than 3.9
+if ! python -c 'import sys; assert sys.version_info >= (3,9)' > /dev/null; then
+    exit "Python should be 3.9 or later!"
+fi
+
+# Clone repo
+git clone https://github.com/intel/llm-on-ray.git && cd llm-on-ray
+
+# Install dependencies
+pip install .[cpu] -f https://developer.intel.com/ipex-whl-stable-cpu -f https://download.pytorch.org/whl/torch_stable.html
+
+# Dynamic link oneCCL and Intel MPI libraries
+source $(python -c "import oneccl_bindings_for_pytorch as torch_ccl;print(torch_ccl.cwd)")/env/setvars.sh
+
+# Start Ray
+ray start --head
+
+# Step 2: Serving
+# take gpt2 for example
+python ../inference/serve.py --config_file inference/models/gpt2.yaml
```

Comment:
we have updated the way to start serving by using llm_on_ray-serve, please check README.md for details.